backend/Squirrel.Core/Squirrel.Core.WebAPI/Validators/Project/UpdateProjectDtoValidator.cs

```diff
@@ -0,0 +1,17 @@
+﻿using FluentValidation;
+using Squirrel.Core.Common.DTO.Project;
+using System.Text.RegularExpressions;
+
+namespace Squirrel.Core.WebAPI.Validators.Project;
+
+public class UpdateProjectDtoValidator: AbstractValidator<UpdateProjectDto>
+{
+    public UpdateProjectDtoValidator()
+    {
+        RuleFor(x => x.Description)!.MaximumLength(1000);
+        RuleFor(x => x.Name)!
+            .MinimumLength(3)!
+            .MaximumLength(50)
+            .Must(x => Regex.IsMatch(x, @"^(?![\u0410-\u044F\u0400-\u04FF]).*$"));
```

Comment:
this is the same regex for the project name, so make a const field for it and use everywhere